### PR TITLE
03896 src directory move v2

### PR DESCRIPTION
**Pull request by mhess-swl**
**Description**:

In preparation for a large refactoring effort (see #3893), this PR moves the `hedera-node` code into a subproject
* Move `src` and `data` directories from `hedera-node` to `hedera-node/hedera-mono-service/`
* Move `settings.txt` and `config.txt` to `hedera-mono-service`
* Transfer logic from `hedera-node` build file to `hedera-mono-service` build file
* ~~Add support for environment variables~~ will put in a new PR
* Fix affected tests
* Update code copyright notices and .gitignore

**Related issue(s)**:

Majority of the work for #3896

### CODE REVIEW

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java [hunk 1]
```diff
@@ -56,12 +57,37 @@ public final class BootstrapProperties implements PropertySource {
...
-    private static ThrowingStreamProvider fileStreamProvider =
+    static final ThrowingStreamProvider DEFAULT_FILE_STREAM_PROVIDER =
             loc -> Files.newInputStream(Paths.get(loc));
 
+    private final EnvironmentProvider envVarsProvider;
+
     @Inject
     public BootstrapProperties() {
-        /* No-op */
+        this(
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java:67]: I think these changes should be separated from this PR. Once this PR is merged a new PR can be opened for these changes.

**Comment by mhess-swl** [comment]
Per feedback from @Neeharika-Sompalli I've reverted the environment var changes, and will put them in a new PR

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/initialization/HfsSystemFilesManagerTest.java [hunk 2]
```diff
@@ -99,7 +99,8 @@
 class HfsSystemFilesManagerTest {
     private static final String R4_FEE_SCHEDULE_REPR_PATH =
             "src/test/resources/testfiles/r4FeeSchedule.bin";
-    private static final String bootstrapJutilPropsLoc = "src/test/resources/bootstrap.properties";
+    private static final String bootstrapJutilPropsLoc =
+            "src/test/resources/bootstrap/hfsSystemFilesBootstrap.properties";
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/initialization/HfsSystemFilesManagerTest.java:103]: This is intentional ?

Comment (mhess-swl) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/initialization/HfsSystemFilesManagerTest.java:103]: Yes, it was conflicting with the new `bootstrap.properties` file I added that fixed the `ServicesState` tests

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! We might need to inform everyone in the channel, since every opened PR will have conflicts after this merge.

**Review by kimbor — COMMENTED** [review]
lgtm

**Review by nathanklick — APPROVED** [review]

**Comment by mhess-swl** [comment]
Note: environment var code has been removed from this PR and instead placed in #4036

**Pull request merged**